src/libraries/Microsoft.PowerFx.Interpreter/Marshal/ObjectMarshaller.cs

```diff
@@ -14,31 +18,26 @@ namespace Microsoft.PowerFx
     [DebuggerDisplay("ObjMarshal({Type})")]
     public class ObjectMarshaller : ITypeMarshaller
```

Comment:
Open question here:
Should we always default to lazy marshalling for C# objects? There's a memory/perf cost to creating all those callbacks.

Reply:
Useful to do for this PR, since it allows our marshalling tests to catch a bunch of cases.

Reply:
It would be good if we were always lazy.  I'm concerned a bifurcation where we have lazy vs. non lazy. Can we find a way to be efficient about it?